paciente_sumarYsips/controller/consultas.ts

```diff
@@ -0,0 +1,525 @@
+
+import { log } from '@andes/log';
+import * as moment from 'moment';
+import * as sql from 'mssql';
+import * as operaciones from '../service/operaciones.service';
+
+let fakeRequest = {
+        user: {
+                usuario: 'sipsYsumar',
+                app: 'integracion-sipsYsumar',
+                organizacion: 'sss'
+        },
+        ip: 'localhost',
+        connection: {
+                localAddress: ''
+        }
+};
+export async function getEfector(codigoCuie, conexion) {
+        if (codigoCuie) {
+                const query = `SELECT [idEfector]
+                ,[nombre]
+                ,[codigoSisa]
+            FROM [dbo].[Sys_Efector] where cuie='${codigoCuie}'`;
+
+                const result = await conexion.request().query(query);
+                if (result && result.recordset) {
+                        return result.recordset[0].idEfector;
+                }
+
+        }
+
+}
+export async function getProvincia(codIndec, conexion) {
+        if (codIndec) {
+                const query = `SELECT [idProvincia]
+            FROM [dbo].[Sys_Provincia] where codigoINDEC = ${codIndec}`;
+
+                const result = await conexion.request().query(query);
+                if (result && result.recordset) {
+                        return result.recordset[0].idProvincia;
+                }
+
+        }
+
+}
+export async function getObraSocial(codigoPuco, conexion) {
+        if (codigoPuco) {
+                const query = `SELECT [idObraSocial],[nombre]
+                FROM [dbo].[Sys_ObraSocial] where cod_PUCO ='${codigoPuco}'`;
+                const result = await conexion.request().query(query);
+                if (result && result.recordset) {
+                        return result.recordset[0].idObraSocial;
+                }
+        }
+
+}
+export async function getDepartamento(nombreDpto, conexion) {
+        if (nombreDpto) {
+                const query = `SELECT [idDepartamento]
+                ,[nombre]
+            FROM  [dbo].[Sys_Departamento] WHERE nombre= '${nombreDpto}'`;
+                const result = await conexion.request().query(query);
+                if (result && result.recordset) {
+                        return result.recordset[0].idDepartamento;
+                }
+        }
+
+}
+
+export async function existePacienteSIPS(paciente: any, conexion) {
+        const dni = parseInt(paciente.documento, 10);
+        if (dni) {
+                const query = `SELECT TOP 1 *
+            FROM [dbo].[Sys_Paciente] where [numeroDocumento] = '${dni}'`;
```

Comment:
Aqui habria que filtrar solo los pacientes que estan activos
```suggestion
            FROM [dbo].[Sys_Paciente] where [activo]=1 and [numeroDocumento] = '${dni}'`;
```